app/src/lib/get-os.ts

```diff
@@ -16,3 +16,24 @@ export function getOS() {
     return `${OS.type()} ${OS.release()}`
   }
 }
+
+/** See the OS we're currently running on is at least Mojave. */
+export function isMojaveOrLater() {
+  if (__DARWIN__) {
+    const parser = new UAParser()
+    const os = parser.getOS()
+
+    if (os.version == null) {
+      return false
+    }
+
+    const parts = os.version.split('.')
```

Comment:
I think we can be more descriptive here by pulling these out into variables. `const [major, minor] = os.version.split('.')` Doing it this way makes it clear what each part of the version means.